docs/cvl/v5-changes.md

```diff
@@ -193,6 +194,52 @@ is the main contract being verified, then passing `--contract C` will cause
 method variables to be instantiated in the same way the would have in older
 versions.
 
+Disallow Solidity function calls in quantified expressions
+----------------------------------------------------------
+
+Starting with `certora-cli` version 5.0, the Prover no longer supports
+making Solidity contract method calls in quantified expression bodies by
+default. For example, given the simple contract below, you can no longer
+use the `method` `foo()` in a quantified expression body.
+
+```{code-block} solidity
+contract example {
+   function foo(uint256 i) public pure returns (uint256) {
+       return i;
+   } 
+}
+```
+
+```{code-block} cvl
+:emphasize-lines: 4
+
+rule for_all() {
+    // Using foo(i) in the quantified body will now cause the prover to
+    // generate a type-checking error.
+    require (forall uint256 i . i == foo(i));
+    assert false, "Prover will generate an error before this line";
+}
+```
+
+In the example rule `for_all`, the prover will now generate an error similar
```

Comment:
```suggestion
In the example rule `for_all`, the Prover will now generate an error similar
```

Reply:
I'm surprised this passed the spell checker - somebody must have added lowercase-p prover to the dictionary.  I'll remove it

Reply:
Uppercased.